### PR TITLE
Springboot/show iex quote

### PR DESCRIPTION
**Pull request by AkshaySapra**
This PR involves tickets upto "Quote Table" and partially "QuoteController"
https://www.notion.so/QuoteController-3e08dc525f2943238e9668e82b127b02
https://www.notion.so/Quote-Table-b43f399f45584feca34039681524bb21
https://www.notion.so/Show-IexQuote-fec7fce447e34913a9ae9731243d8567
https://www.notion.so/Architecture-Design-2a95dde5fcd14109a3d65a1ba8ea1513

### CODE REVIEW

**Comment by AkshaySapra** [comment]
I realize that this pull request is fairly large but doesn't have many commits. I forgot to commit frequently

File: springboot/api-docs.json [hunk 1]
```diff
@@ -0,0 +1 @@
...,"format":"int32"},"bidPrice":{"type":"number","format":"double"},"bidSize":{"type":"integer","format":"int32"},"id":{"type":"string"},"lastPrice":{"type":"number","format":"double"},"ticker":{"type":"string"}},"title":"Quote"},"SecurityOrder":{"type":"object","properties":{"accountId":{"type":"integer","format":"int32"},"id":{"type":"integer","format":"int32"},"notes":{"type":"string"},"price":{"type":"number","format":"double"},"size":{"type":"integer","format":"int32"},"status":{"type":"string","enum":["FILLED","CANCELED","PENDING"]},"ticker":{"type":"string"}},"title":"SecurityOrder"},"SecurityRow":{"type":"object","properties":{"position":{"$ref":"#/definitions/Position"},"quote":{"$ref":"#/definitions/Quote"},"ticker":{"type":"string"}},"title":"SecurityRow"},"Trader":{"type":"object","properties":{"country":{"type":"string"},"dob":{"type":"string","format":"date-time"},"email":{"type":"string"},"firstName":{"type":"string"},"id":{"type":"integer","format":"int32"},"lastName":{"type":"string"}},"title":"Trader"},"TraderAccountView":{"type":"object","properties":{"account":{"$ref":"#/definitions/Account"},"trader":{"$ref":"#/definitions/Trader"}},"title":"TraderAccountView"}}}
```
Comment (jarviscanada) [inline springboot/api-docs.json:1]: remove this as it's a generated file (by swagger)

File: springboot/src/main/java/ca/jrvs/apps/trading/AppConfig.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        marketDataConfig.setToken(System.getenv("IEX_PUB_TOKEN"));
+        return marketDataConfig;
+    }
+
+    @Bean
+    public PoolingHttpClientConnectionManager poolingHttpClientConnectionManager() {
+        PoolingHttpClientConnectionManager cm = new PoolingHttpClientConnectionManager();
+        cm.setMaxTotal(50);
+        cm.setDefaultMaxPerRoute(50);
+        return cm;
```
Comment (jarviscanada) [inline springboot/src/main/java/ca/jrvs/apps/trading/AppConfig.java:31]: make sure you understand pooling

File: springboot/trading.iml [hunk 3]
```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
+  <component name="FacetManager">
+    <facet type="jpa" name="JPA">
+      <configuration>
+        <setting name="validation-enabled" value="true" />
+        <setting name="provider-name" value="Hibernate" />
```
Comment (jarviscanada) [inline springboot/trading.iml:7]: remove intellij generated file

**Review by jarviscanada — APPROVED** [review]

**Pull request merged**